CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ elseif(WIN32)
   include(Win32Configuration)
 elseif(UNIX AND (NOT APPLE))
   include(LinuxConfiguration)
+  include(InstallDesktopFiles)
```

Comment:
For clarity it would probably bettter that this gets into the LinixConfiguration cmake file like for other platforms.

Reply:
Hm, but than we'd have to have an `if(NOT OPENELEC)` for installing the desktop files, which would be rather confusing in the LinuxConfiguration file (since OpenELEC is Linux as well)